src/lib/ext-monitor.js

```diff
@@ -97,7 +123,15 @@ export default class ExtensionMonitor {
     }
   };
 
+  onRemovedListener = (tabId) => {
+    // When we close any tab which was loaded with logs from a log file.
```

Comment:
Nit, `// Remove the loaded logs related to the closed tab` may be slightly more clear and as short as yours.

Reply:
done :-)